app/controllers/state_file/questions/az_public_school_contributions_controller.rb

```diff
@@ -65,6 +67,11 @@ def destroy
 
       private
 
+      def has_valid_cents?
+        amount = az322_contribution_params[:amount]
+        amount.to_s.match?(/^(\d+)?\.?\d{0,2}$/)
```

Comment:
played around with this https://regex101.com/r/fUi2Wb/1 to figure out regex for what we want to consider "valid"
![Screenshot 2024-09-20 at 2 12 41 PM](https://github.com/user-attachments/assets/11d048e3-76cb-4628-9031-24052278f7ed)